debugger.md

```diff
@@ -140,6 +140,18 @@ The target process can optionally launch into a separate console window. You wil
 
     "externalConsole": true
 
+##### Stepping into properties and operators
+The debugger steps over properties and operators in managed code by default. In most cases, this provides a better debugging experience. To change this and enable stepping into properties or operators add:
+
+    "enableStepFiltering": false
+
+#### Attach Support
+The C# debugger supports attaching to processes. To do this, switch to the Debug tab, and open the configuration drop down.
+
+![Debug launch configuration drop down](https://raw.githubusercontent.com/wiki/OmniSharp/omnisharp-vscode/images/debug-launch-configurations.png)
+
+Select the '.NET Core Attach' configuration. Hitting the play button (or pressing F5) will then try and attach. In launch.json, if `processId` is set to `"${command.pickProcess}"` this will provide UI to select which process to attach to.
```

Comment:
"Clicking" rather than "Hitting"?

"try to attach" rather than "try and attach"